README.md

```diff
@@ -506,6 +506,32 @@ Decimal('1.23456789E-10')
 ```
 
 
+### Debug Utils
+
+#### Generate environment info
+
+At the shell:
```

Comment:
???

> In your shell

Reply:
Haha, your question is "why?" I take it?

I want to add something to the issue template, like "run this and copy in the output:"

```sh
python -m eth_utils
```

I think we'll get better compliance that way, than saying something like:

```sh
python
>>> import eth_utils
>>> eth_utils.get_environment_summary()
```

---

Totally not married to this approach, though. Any suggestions for other ways to make this easy for people?

Reply:
Lol, I was merely confused by the sentence "At the shell".  The wording seemed *minorly* weird.  Disregard if you don't see it.

-  "I am at my shell"
- "I typed a command **in** my shell"
- "I typed a command *at* my shell????"